src/store/modules/example/reducer.ts

```diff
@@ -0,0 +1,16 @@
+import { Action } from "./action";
+import { EXAMPLE_REQUEST_TYPES } from "./types";
+
+export const exampleReducer = (state = 0, action: Action) => {
+  switch (action.type) {
+    case EXAMPLE_REQUEST_TYPES.EXAMPLE_INCREAMENT_COUNTER:
+      return state + 1;
```

Comment:
This is not the right way to update the state in the reducer.

Reply:
resolved it in correct way